app/src/main/kotlin/no/nav/k9punsj/journalpost/JournalpostRoutes.kt

```diff
@@ -259,7 +259,7 @@ internal class JournalpostRoutes(
                         .notFound()
                         .buildAndAwait()
 
-                aksjonspunktService.settUtførtPåAltSendLukkOppgaveTilK9Los(journalpostId, false)
```

Comment:
Når brukes dette endepunktet? Er dette en utført punsjeoppgave?

Reply:
https://github.com/navikt/k9-punsj/blob/53923166dfeccae04a83f4e2b92944904f78f9de/app/src/main/kotlin/no/nav/k9punsj/fordel/HendelseMottaker.kt#L58

Om vi mottar journalpost från fordel som redan finnes i k9-punsj & isåfall om typen er PUNSJOPPGAVE_IKKE_LENGER_NØDVENDIG :)